clang on powerpc64 defines vec_step already which collides with the
symbol in gcc, see https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=239266#c23
renaming suggested at https://gcc.gnu.org/legacy-ml/gcc/2019-07/msg00131.html

Index: sources/tools/gcc/tree-vect-loop.c
--- sources/tools/gcc/tree-vect-loop.c.orig
+++ sources/tools/gcc/tree-vect-loop.c
@@ -56,6 +56,8 @@ along with GCC; see the file COPYING3.  If not see
 #include "tree-vectorizer.h"
 #include "target.h"
 
+#define vec_step vec_step_
+
 /* Loop Vectorization Pass.
 
    This pass tries to vectorize loops.
